roles/frontend/templates/nginx-default.d-webroot.conf.j2

```diff
@@ -1,9 +1,32 @@
 location / {
+    try_files $uri @prerender;
```

Comment:
We also don't need this - the @prerender block should be unconditionally used for / - there is no meaningful file root set for the webroot.

Reply:
This I don't know. I just copied from a gist and it works fine